base/range.jl

```diff
@@ -860,13 +860,18 @@ end
 
 median(r::Range{<:Real}) = mean(r)
 
-function in(x, r::Range)
+function _in_range(x, r::Range)
     n = step(r) == 0 ? 1 : round(Integer,(x-first(r))/step(r))+1
```

Comment:
The tangential discussion re. the potential type instability in this line got buried above. Any thoughts re. addressing this type instability while you're performing surgery? :)